docs/PROCESSORS_TO_REFACTOR.md

```
@@ -238,8 +238,8 @@ All 15 identified processors have been successfully refactored to use `PublishCh
 
 #### Medium Priority ✅
 2. EventDocumentProcessor - 4 child document requests refactored
-3. AthleteDocumentProcessor - 1 dependency request refactored
-4. CoachBySeasonDocumentProcessor - 2 dependency requests refactored
+3. FootballAthleteDocumentProcessor - 1 dependency request refactored
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)